src/main/kotlin/no/nav/tilleggsstonader/kontrakter/søknad/læremidler/AvsnittLæremidler.kt

```diff
@@ -7,7 +7,7 @@ import no.nav.tilleggsstonader.kontrakter.søknad.JaNei
 data class UtdanningAvsnitt(
     val aktiviteter: EnumFlereValgFelt<String>? = null,
     val annenUtdanning: EnumFelt<AnnenUtdanningType>? = null,
-    val mottarUtstyrsstipend: EnumFelt<JaNei>? = null,
+    val erLærligEtc: EnumFelt<JaNei>? = null,
```

Comment:
```suggestion
    val erLærlingEtc: EnumFelt<JaNei>? = null,
```
`etc` føles litt uheldig 😬 

Er spørsmålet kun koblet til videregående utdanning?

Kanskje `erLærlingEllerLiknende` eller `utdanningMedPraksis` ?

Reply:
Det skal være et samlebegrep for 'lærling, lærekandidat, praksisbrevkandidat eller kandidat for fagbrev på jobb'. Vi diskuterte det litt i går og fant ikke noe godt begrep. Kan godt skrive det og bruke 'erLærligEllerLiknende'.